Makefile

```diff
@@ -0,0 +1,90 @@
+SHELL := /bin/bash
+
+# Replace these with the name and domain of your extension!
+NAME     := bring-out-submenu-of-power-off-logout
+DOMAIN   := pratap.fastmail.fm
+ZIP_NAME := $(NAME)@$(DOMAIN).zip
+
+# Some of the recipes below depend on some of these files.
+JS_FILES       = $(shell find -type f -and \( -name "*.js" \))
+ICONS          = $(shell find -maxdepth 1 -type f -and \( -name "*.svg" \))
```

Comment:
can you automate copy the icons from extensions `icons` directory to user's `.icons` folder of home directory and remove them while uninstalling. Also it will be good to get the confirmation from user for this action. This extension needs two svg icons to be copy pasted into users `.icons` folder of home directory.

Reply:
Please have a look on this link for automating copy pasting of icons https://github.com/PRATAP-KUMAR/bring-out-submenu-of-power-off-logout#optional---hibernation-buttons

Thanks